lib/defaults.js

```diff
@@ -32,6 +32,14 @@ var defaults = module.exports = {
 	'cookie secret': 'oST1Thr2s/iOAUgeK4yuuA==',
 	'date format': 'ddd, ll',
 	'datetime format': 'llll',
-	'set local time': false
-
+	'set local time': false,
+	'permissions': function defaultPermissions (userId, action, context, callback) {
+		console.log('permissions');
+		return callback(null, true);
+	},
+
+	'disabled navigation cache' : false,
+	'disabled permission cache' : false,
+	'disabled config cache' : false
```

Comment:
These should all read _disable_ rather than _disabled_. They'll need to be changed everywhere else too.